tests/test_examples.py

```diff
@@ -7,4 +7,4 @@ def test_pooch_registry():
     This will fail if the remote file does not match the hash hard-coded
     in btrack.datasets.
     """
-    registry_file = datasets._remote_registry()
+    registry_file = datasets._remote_registry()  # noqa: F841
```

Comment:
What's this noqa for? Could you add a comment explaining it for the next person taht comes across it?

Reply:
I'm adding a comment to explain a comment. A Google of `noqa` takes you straight to what it is.

Anyway, it is suppressing `flake8` moaning because of the violation of `F841` https://www.flake8rules.com/rules/F841.html. Basically `registry_file` is created but then not returned/anything happening to it

Reply:
Ideally, someone would implement something and then this comment won't be needed. But it wouldn't be the end of the world if anyone forgot to remove it

Reply:
In that case, can you not just remove `registry_file = ` ?

Reply:
I'm hesitant to, in case whoever made it did it for a reason. Clearly half finished
![image](https://user-images.githubusercontent.com/15052188/167857766-050ece67-73da-492f-bd90-b2179557fe5b.png)